plugins/repository-s3/src/test/java/org/elasticsearch/cloud/aws/AwsS3ServiceImplTests.java

```diff
@@ -38,115 +37,41 @@
 
     public void testAWSCredentialsWithSystemProviders() {
         AWSCredentialsProvider credentialsProvider =
-            InternalAwsS3Service.buildCredentials(logger, deprecationLogger, Settings.EMPTY, Settings.EMPTY);
+            InternalAwsS3Service.buildCredentials(logger, deprecationLogger, Settings.EMPTY, Settings.EMPTY, "default");
         assertThat(credentialsProvider, instanceOf(InstanceProfileCredentialsProvider.class));
     }
 
-    public void testAWSCredentialsWithElasticsearchAwsSettings() {
+    public void testAwsCredsDefaultSettings() {
         Settings repositorySettings = generateRepositorySettings(null, null, "eu-central", null, null);
         MockSecureSettings secureSettings = new MockSecureSettings();
-        secureSettings.setString(AwsS3Service.KEY_SETTING.getKey(), "aws_key");
-        secureSettings.setString(AwsS3Service.SECRET_SETTING.getKey(), "aws_secret");
+        secureSettings.setString("aws.config.default.access_key", "aws_key");
```

Comment:
Don't forget to update to `s3.client.` here. But i'm sure you'll think about it when you will run the tests :) 
I don't comment on other lines in this test but same applies.